### ISSUE TITLE
Erroneous no-extra-semicolons warnings

### ISSUE THREAD

**Issue by cyberkatze** [stylelint/stylelint#1389]
> Describe the issue. Is it a bug or a feature request (new rule, new option, etc.)?

Bug

> Which rule, if any, is this issue related to?

`no-extra-semicolons`

> What CSS is needed to reproduce this issue?

```css
@import "../config.less";

/* ==========================================================================
   jQuery Smartbanner overwrite
   ========================================================================== */
#smartbanner {
    .box-sizing( content-box );
    font-family: @regularFontStack;
    .FontLight;
}
```

> What stylelint configuration is needed to reproduce this issue?

```js
module.exports = {
    "extends": "stylelint-config-standard",
    "rules": {
        "at-rule-name-case": null,
        "at-rule-empty-line-before": "always",
        "block-no-empty": true,
        "color-no-invalid-hex": true,
        "comment-empty-line-before": "always",
        "declaration-colon-space-after": "always",
        "declaration-block-trailing-semicolon": "always",
        "indentation": 4,
        "max-empty-lines": 2,
        "max-line-length": [120,
            {
                ignore: ["non-comments"]
            }
        ],
        "number-leading-zero": "always",
        "selector-pseudo-element-colon-notation":"double",
        "string-quotes": "double",
        "unit-whitelist": [
            "em",
            "rem",
            "px",
            "%"
        ]
    }
};
```

> Which version of stylelint are you using?

```
$ stylelint --version
6.5.1
```

> How are you running stylelint: CLI, PostCSS plugin, Node API?

from CLI with `stylelint -c stylelint.config.js *.less`  and with Gulp-Task:

```js
    gulp.task('lint:css', function() {
        var _styles = devDir + '/styles/**/*.less';
        var processors = [
            stylelint(stylelintConfig),
            reporter({clearMessages: true})
        ];

        return gulp.src(_styles)
            .pipe(postcss(processors, {syntax: postLess}));
    });
```

> Does your issue relate to non-standard syntax (e.g. SCSS, nesting, etc.)?

LESS

> What did you expect to happen?

"No warnings to be flagged."

>  What actually happened (e.g. what warnings or errors you are getting)?

```
overwrite.less
3:73	⚠  Unexpected extra semicolon (no-extra-semicolons) [stylelint]
3:75	⚠  Unexpected extra semicolon (no-extra-semicolons) [stylelint]
7:33	⚠  Unexpected extra semicolon (no-extra-semicolons) [stylelint]
8:27	⚠  Unexpected extra semicolon (no-extra-semicolons) [stylelint]
```

**Comment by davidtheclark** [comment]
@cyberkatze Failing tests or PR greatly appreciated.

**Comment by davidtheclark** [comment]
I think this is an upstream issue caused by https://github.com/webschik/postcss-less/issues/45

**Comment by jeddy3** [comment]
@cyberkatze Please consider helping upstream in [`postcss-less`](https://github.com/webschik/postcss-less/issues/45) if you'd like to see this issue resolved.

**Comment by ntwb** [comment]
@cyberkatze can you please try PostCSS-Less v`0.14.0`?

Change it from `0.13.0` to `0.14.0` in your `package.json` file please
